### PR TITLE
update Stonnington

### PR DESCRIPTION
**Pull request by mnot**
I've tested this locally; it doesn't work on morph.io because the SELECT statement can't find the sqlite file, I think.

### CODE REVIEW

**Comment by henare** [comment]
@mnot awesome! :sparkles: 

@CloCkWeRX have you got time to review this? :crown: Just assign back if you don't.

File: scraper.rb [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-    'council_reference' => r.search('td')[0].inner_text,
-    'date_received' => Date.strptime(r.search('td')[1].inner_text.strip, '%d/%m/%Y').to_s,
-    'address' => r.search('td')[3].inner_text.strip + ", VIC",
-    'description' => r.search('td')[4].inner_text.strip,
-    'date_scraped' => Date.today.to_s
+    'date_scraped' => Date.today.to_s,
+    'date_received' => Date.strptime(r['LodgedDate_STRING'], "%d-%b-%Y").to_s
   }
 
   if ScraperWiki.select("* from data where `council_reference`='#{record['council_reference']}'").empty?
```
Comment (CloCkWeRX) [inline scraper.rb]: OK to chuck a rescue true in here, it creates db on save below

**Comment by CloCkWeRX** [comment]
https://morph.io/CloCkWeRX/stonnington - the addresses aren't super great, but I suppose we can't do much about that!

**Comment by CloCkWeRX** [comment]
Any chance you feel like upping the 
```
{"name":"iDisplayLength","value":10}
```
to 25? 
Or having a crack at pagination? 10 per day is probably enough not to miss any, but more data is better!

**Comment by CloCkWeRX** [comment]
Happy with this, fixes #2

**Comment by mnot** [comment]
Increasing `iDisplayLength` seems sensible to me too.

No idea what "chucking a rescue true" is -- I'm not a Ruby person :)

**Comment by henare** [comment]
Woohoo :tada: thanks for this @mnot and thanks for reviewing @CloCkWeRX!

@LoveMyData said this in our Slack chat:

>@mnot, @clockwerx: clever fix on stonnington planningalerts, thx, learn something new from your code….

That's great :smile: 

>No idea what "chucking a rescue true" is -- I'm not a Ruby person :)

@mnot It's a hack to ignore the exception that's thrown when there's no database the first time the scraper runs. @CloCkWeRX [has already added it](https://github.com/planningalerts-scrapers/stonnington/commit/78fae789b6f186fcd1a49d1bc1150cdc984410ca).

**Pull request merged**